.github/build-mandrel-images.java

```diff
@@ -0,0 +1,179 @@
+//usr/bin/env jbang "$0" "$@" ; exit $?
+//DEPS info.picocli:picocli:4.5.0
+//DEPS org.eclipse.jgit:org.eclipse.jgit:5.8.1.202007141445-r
+//DEPS org.zeroturnaround:zt-exec:1.11
+//DEPS com.github.docker-java:docker-java:3.2.5
+//DEPS com.github.docker-java:docker-java-transport-okhttp:3.2.5
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.model.Image;
+import com.github.dockerjava.api.model.PruneType;
+import com.github.dockerjava.core.DefaultDockerClientConfig;
+import com.github.dockerjava.core.DockerClientConfig;
+import com.github.dockerjava.core.DockerClientImpl;
+import com.github.dockerjava.okhttp.OkDockerHttpClient;
+import com.github.dockerjava.transport.DockerHttpClient;
+import com.google.common.collect.ImmutableMap;
+import org.zeroturnaround.exec.ProcessExecutor;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+import java.io.File;
+import java.util.*;
+import java.util.concurrent.Callable;
+import java.util.stream.Collectors;
+
+@Command(name = "build-mandrel-images", mixinStandardHelpOptions = true,
+        description = "build container images providing the `native-image` executable from Mandrel")
+class BuildMandrelImage implements Callable<Integer> {
+
+    static final List<String> VERSIONS = Arrays.asList("20.1.0.0.Alpha1-java11", "20.1.0.1.Alpha2-java11", "20.1.0.1.Final-java11");
+    static final Map<String, String> TAGS = ImmutableMap.of(
+            "20.1-java11", "20.1.0.1.Final-java11"
```

Comment:
We should automate this part so that we can just add `VERSIONS` and get major.minor `TAGS` automatically created.
(I can help with that)

Reply:
I wanted to avoid this. For Quarkus we want to be a fine control on the tags and not update to the latest automatically. 

This is one of the reasons for the duplication because there is no easy way to pass that data to the script.

Reply:
> I wanted to avoid this. For Quarkus we want to be a fine control on the tags and not update to the latest automatically.

I see. So we first get the new images out, we test them and if all looks well we bump the floating tag.

> This is one of the reasons for the duplication because there is no easy way to pass that data to the script.

We can change `VERSIONS` type to  `Map<String, List<String>>` and `TAGS` to `Map<String, Map<String, String>` and use `mandrel` and `graalvm` as keys. Then we can pass the "flavour" (mandrel or graalvm) to the script.
WDYT?

Reply:
We could even have these kind of info in some `.properties`/`.config` and load them from there.

Reply:
Yes, that would be nicer!

Reply:
Would you like me to try this out and open a PR against your branch or are you planning to work on it yourself?

Reply:
That would be awesome!

Reply:
Have a look at https://github.com/cescoffier/quarkus-images/pull/1 later (tests are still running :) )